modules/build/src/main/scala/scala/build/errors/NetworkUnaccessibleScalaVersionError.scala

```diff
@@ -0,0 +1,6 @@
+package scala.build.errors
+
+final class NetworkUnaccessibleScalaVersionError(
```

Comment:
I'm not sure that error makes sense… If there are network issues, downloading the version listing will just fail, and we'll get an error from coursier earlier.

Reply:
Thanks, so I would remove it.